src/util/getDiff.ts

```diff
@@ -1,26 +1,68 @@
 import { assert } from "../util/assert";
+import { JimpClass } from "@jimp/types";
+import { Jimp } from "jimp";
+import { methods } from "@jimp/plugin-resize";
+import { clone } from "@jimp/utils";
 import pixelMatch from "pixelmatch";
-import { PNG } from "pngjs/browser";
 
-export function getDiff(aPng: PNG, bPng: PNG) {
-  assert(aPng.width === bPng.width && aPng.height === bPng.height);
-  const diff = new PNG({ width: aPng.width, height: bPng.height });
-  const diffPixelCount = pixelMatch(
-    aPng.data,
-    bPng.data,
-    diff.data,
-    aPng.width,
-    aPng.height,
+export function diff<I extends JimpClass>(img1: I, img2: I) {
+  let bmp1 = img1.bitmap;
+  let bmp2 = img2.bitmap;
+
+  if (bmp1.width !== bmp2.width || bmp1.height !== bmp2.height) {
```

Comment:
If you only use this function in `getDiff` then this should never happen - there's an assertion that both are the same dimensions.

Reply:
okk